<robot
  name="PONGBOT_Q_V2">
  <link
    name="REAR_BODY">
    <inertial>
      <origin
        xyz="0.1216 0.0 0.0201"
        rpy="0 0 0" />
      <mass
        value="7.7996" />
      <inertia
        ixx="0.0745"
        ixy="0.0"
        ixz="-0.0"
        iyy="0.1354"
        iyz="-0.0"
        izz="0.1812" />
    </inertial>
  </link>
  <link
    name="RR_HIP">
    <inertial>
      <origin
        xyz="-0.0009 0.0001 -0.0002"
        rpy="0 0 0" />
      <mass
        value="1.1163" />
      <inertia
        ixx="0.0011"
        ixy="0"
        ixz="0"
        iyy="0.0012"
        iyz="0"
        izz="0.0012" />
    </inertial>
  </link>
  <joint
    name="RR_HIP_JOINT"
    type="continuous">
    <origin
      xyz="0 -0.11502 -0.00031465"
      rpy="0 0 0" />
    <parent
      link="REAR_BODY" />
    <child
      link="RR_HIP" />
    <axis
      xyz="1 0 0" />
  </joint>
  <link
    name="RR_THIGH">
    <inertial>
      <origin
        xyz="0 0.0163 -0.0263"
        rpy="0 0 0" />
      <mass
        value="1.8748" />
      <inertia
        ixx="0.0112"
        ixy="0"
        ixz="0"
        iyy="0.0117"
        iyz="0.0"
        izz="0.0017" />
    </inertial>
  </link>
  <joint
    name="RR_THIGH_JOINT"
    type="continuous">
    <origin
      xyz="0 -0.105 -0.00028723"
      rpy="0 0 0" />
    <parent
      link="RR_HIP" />
    <child
      link="RR_THIGH" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="RR_CALF">
    <inertial>
      <origin
        xyz="0.0012 -0.0017 -0.0897"
        rpy="0 0 0" />
      <mass
        value="0.402" />
      <inertia
        ixx="0.0052"
        ixy="0"
        ixz="-0.0"
        iyy="0.0051"
        iyz="-0.0"
        izz="0.0002" />
    </inertial>
  </link>
  <joint
    name="RR_CALF_JOINT"
    type="continuous">
    <origin
      xyz="0 0.00083432 -0.305"
      rpy="0 0 0" />
    <parent
      link="RR_THIGH" />
    <child
      link="RR_CALF" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="RL_HIP">
    <inertial>
      <origin
        xyz="-0.0009 -0.0001 0.0002"
        rpy="0 0 0" />
      <mass
        value="1.1163" />
      <inertia
        ixx="0.0011"
        ixy="0"
        ixz="0"
        iyy="0.0012"
        iyz="0"
        izz="0.0012" />
    </inertial>
  </link>
  <joint
    name="RL_HIP_JOINT"
    type="continuous">
    <origin
      xyz="0 0.11502 -0.00031465"
      rpy="0 0 0" />
    <parent
      link="REAR_BODY" />
    <child
      link="RL_HIP" />
    <axis
      xyz="1 0 0" />
  </joint>
  <link
    name="RL_THIGH">
    <inertial>
      <origin
        xyz="0 -0.0163 -0.0263"
        rpy="0 0 0" />
      <mass
        value="1.8748" />
      <inertia
        ixx="0.0112"
        ixy="0"
        ixz="0"
        iyy="0.0117"
        iyz="-0.0"
        izz="0.0017" />
    </inertial>
  </link>
  <joint
    name="RL_THIGH_JOINT"
    type="continuous">
    <origin
      xyz="0 0.105 0.00028723"
      rpy="0 0 0" />
    <parent
      link="RL_HIP" />
    <child
      link="RL_THIGH" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="RL_CALF">
    <inertial>
      <origin
        xyz="0.0012 0.0017 -0.0897"
        rpy="0 0 0" />
      <mass
        value="0.402" />
      <inertia
        ixx="0.0052"
        ixy="0"
        ixz="-0.0"
        iyy="0.0051"
        iyz="0"
        izz="0.0002" />
    </inertial>
  </link>
  <joint
    name="RL_CALF_JOINT"
    type="continuous">
    <origin
      xyz="0 0.00083432 -0.305"
      rpy="0 0 0" />
    <parent
      link="RL_THIGH" />
    <child
      link="RL_CALF" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="FRONT_BODY">
    <inertial>
      <origin
        xyz="0.2266 -0.0 0.0015"
        rpy="0 0 0" />
      <mass
        value="8.9472" />
      <inertia
        ixx="0.0895"
        ixy="0.0"
        ixz="-0.0"
        iyy="0.1228"
        iyz="0"
        izz="0.1674" />
    </inertial>
  </link>
  <joint
    name="WAIST_JOINT"
    type="continuous">
    <origin
      xyz="0.35 0 0.035238"
      rpy="0 0 0" />
    <parent
      link="REAR_BODY" />
    <child
      link="FRONT_BODY" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="FR_HIP">
    <inertial>
      <origin
        xyz="0.0009 -0.0001 -0.0002"
        rpy="0 0 0" />
      <mass
        value="1.1163" />
      <inertia
        ixx="0.0011"
        ixy="0"
        ixz="0"
        iyy="0.0012"
        iyz="0"
        izz="0.0012" />
    </inertial>
  </link>
  <joint
    name="FR_HIP_JOINT"
    type="continuous">
    <origin
      xyz="0.35 -0.11502 -0.03555"
      rpy="0 0 0" />
    <parent
      link="FRONT_BODY" />
    <child
      link="FR_HIP" />
    <axis
      xyz="1 0 0" />
  </joint>
  <link
    name="FR_THIGH">
    <inertial>
      <origin
        xyz="0 0.0163 -0.0263"
        rpy="0 0 0" />
      <mass
        value="1.8748" />
      <inertia
        ixx="0.0112"
        ixy="0"
        ixz="0"
        iyy="0.0117"
        iyz="0.0"
        izz="0.0017" />
    </inertial>
  </link>
  <joint
    name="FR_THIGH_JOINT"
    type="continuous">
    <origin
      xyz="0 -0.105 0"
      rpy="0 0 0" />
    <parent
      link="FR_HIP" />
    <child
      link="FR_THIGH" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="FR_CALF">
    <inertial>
      <origin
        xyz="0.0012 -0.0017 -0.0897"
        rpy="0 0 0" />
      <mass
        value="0.402" />
      <inertia
        ixx="0.0052"
        ixy="0"
        ixz="0.0"
        iyy="0.0051"
        iyz="0"
        izz="0.0002" />
    </inertial>
  </link>
  <joint
    name="FR_CALF_JOINT"
    type="continuous">
    <origin
      xyz="0 0 -0.305"
      rpy="0 0 0" />
    <parent
      link="FR_THIGH" />
    <child
      link="FR_CALF" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="FL_HIP">
    <inertial>
      <origin
        xyz="0.0009 0.0001 0.0002"
        rpy="0 0 0" />
      <mass
        value="1.1163" />
      <inertia
        ixx="0.0011"
        ixy="0"
        ixz="0"
        iyy="0.0012"
        iyz="0"
        izz="0.0012" />
    </inertial>
  </link>
  <joint
    name="FL_HIP_JOINT"
    type="continuous">
    <origin
      xyz="0.35 0.11502 -0.03555"
      rpy="0 0 0" />
    <parent
      link="FRONT_BODY" />
    <child
      link="FL_HIP" />
    <axis
      xyz="1 0 0" />
  </joint>
  <link
    name="FL_THIGH">
    <inertial>
      <origin
        xyz="0 -0.0163 -0.0263"
        rpy="0 0 0" />
      <mass
        value="1.8748" />
      <inertia
        ixx="0.0112"
        ixy="0"
        ixz="0"
        iyy="0.0117"
        iyz="-0.0"
        izz="0.0017" />
    </inertial>
  </link>
  <joint
    name="FL_THIGH_JOINT"
    type="continuous">
    <origin
      xyz="0 0.105 0"
      rpy="0 0 0" />
    <parent
      link="FL_HIP" />
    <child
      link="FL_THIGH" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="FL_CALF">
    <inertial>
      <origin
        xyz="0.0012 0.0017 -0.0897"
        rpy="0 0 0" />
      <mass
        value="0.402" />
      <inertia
        ixx="0.0052"
        ixy="0"
        ixz="0.0"
        iyy="0.0051"
        iyz="0"
        izz="0.0002" />
    </inertial>
  </link>
  <joint
    name="FL_CALF_JOINT"
    type="continuous">
    <origin
      xyz="0 0 -0.305"
      rpy="0 0 0" />
    <parent
      link="FL_THIGH" />
    <child
      link="FL_CALF" />
    <axis
      xyz="0 1 0" />
  </joint>
</robot>
